### PR TITLE
[Onboarding] - Groundwork for all use cases

### PR DESCRIPTION
**Pull request by antonjoel82**
### What change does this PR introduce?

- Add all static links for each use case
- Connect In-App tab to specific Integration store side panel
- Create abstraction to make adding animations easier for the different color schemes
- Fix Timeline styling in light mode

### Why was this change needed?

- Reduces future work while we await next steps (for [NV-3243](https://linear.app/novu/issue/NV-3243/🙋-in-app-use-case-tab) - 3247)

### Other information (Screenshots)

https://www.loom.com/share/1d42b72c833e4f128353721d0cc420b3?sid=6fde9997-ecb6-4bd0-806a-a2bacf1d76b0

### CODE REVIEW

File: apps/web/src/pages/get-started/consts/DelayUseCase.const.tsx [hunk 1]
```diff
@@ -25,7 +27,7 @@ export const DelayUseCaseConst: OnboardingUseCase = {
         return (
           <StepDescription>
             <StepText>Novu pre-built workflow with a digest node.</StepText>
-            <Link children={' Customize '} href={'https://mantine.dev/core/timeline/'} />
+            <Link href={ROUTES.INTEGRATIONS}> Integration store</Link>
```
Comment (LetItRock) [inline apps/web/src/pages/get-started/consts/DelayUseCase.const.tsx:30]: The value here should be actually `Customize`, [ref](https://www.figma.com/file/hlsUxjq2kxUU7aVBf2QyGu/Onboarding--experience?node-id=3608%3A25001&mode=dev)
But the link (button) would call the API endpoint to create the workflow from the blueprint constant trigger id.

File: apps/web/src/pages/get-started/consts/InAppUseCase.const.tsx [hunk 2]
```diff
@@ -10,9 +15,20 @@ export const InAppUseCaseConst: OnboardingUseCase = {
     {
       title: 'Configure In-App provider',
       Description: function () {
+        const { integrations } = useGetIntegrationsByChannel({ channelType: ChannelTypeEnum.IN_APP });
+
+        const getInAppIntegrationUrl = () => {
+          const inAppIntegration = integrations?.[0];
+          if (!inAppIntegration) {
+            return ROUTES.INTEGRATIONS;
```
Comment (LetItRock) [inline apps/web/src/pages/get-started/consts/InAppUseCase.const.tsx:23]: in this case, I think we can open the Select provider sidebar with a preselected In-App tab... this URL should do that `/integrations/create?scrollTo=in_app`

Comment (antonjoel82) [reply apps/web/src/pages/get-started/consts/InAppUseCase.const.tsx:23]: Sounds good, thanks!

File: apps/web/src/pages/get-started/consts/DelayUseCase.const.tsx (hunk 1 above)
Comment (antonjoel82) [reply apps/web/src/pages/get-started/consts/DelayUseCase.const.tsx:30]: Shoot thank you, good catch

**Review by LetItRock — APPROVED** [review]
🔥

**Pull request merged**